app/Http/Controllers/AdminEditController.php

```diff
@@ -16,15 +16,15 @@ public function index() {
 
     
         public function updateProfile(Request $req) {
-            $name =  Session::get('admin-name');
-            Session::flush();
-            
+            $p = Auth::user()->password;
+            return dd($p);
```

Comment:
what is $p and dd??

Reply:
sir, dd() is a dump and die function. which is used to halt further execution and return value inside it. It is generally used to check for the errors. I mistakenly forgot to remove that.